### PR TITLE
Make inner keyword argument of violin plot accessible from outside

### PR DESCRIPTION
**Pull request by gokceneraslan**
I noticed that there is no way to plot the "box" inside the violins using inner='box' option. This is the main idea of this PR.

### CODE REVIEW

File: scanpy/plotting/_anndata.py [hunk 1]
```diff
@@ -711,6 +711,8 @@ def violin(
     # of the violin plot (see stacked_violin code) for more info.
     if 'cut' not in kwds:
         kwds['cut'] = 0
+    if 'inner' not in kwds:
+        kwds['inner'] = None
```
Comment (ivirshup) [inline scanpy/plotting/_anndata.py]: How about: `kwds.setdefault("inner")`, to skip the `if` statement?

Comment (gokceneraslan) [reply scanpy/plotting/_anndata.py]: Do you mean `kwds.setdefault("inner", None)`?

Comment (gokceneraslan) [reply scanpy/plotting/_anndata.py]: Oh default is None already, I didn't know that. ok, makes sense :)

**Comment by fidelram** [comment]
Thanks for taking care of this. Looks good.

**Pull request merged**